Simulation/TSoG_X1_Sim.m

```diff
@@ -16,83 +16,87 @@
     O0 = [0];     % Initial orientation [Pitch] (degrees)
 
     TestCase.InitialConditions = [P0 V0 O0];
+
     TestCase.PitchTable =  [0 0;
                             5 0;
                             5.001 5;
                             10 5;
                             10.001 0;
                             15 0];
+    % Linear interpolation of Pitch Table
+    TestCase.GetPitch = @(time) interp1(TestCase.PitchTable(:,1),TestCase.PitchTable(:,2),time,TestCase.PitchTable(end,2));
```

Comment:
We could add a function that takes in one of these tables and linearly interpolates it so this code doesn't have to be duplicated in many test cases since it seems like that will be the most common way to handle this data. We can also just make this a separate issue because this is already a big PR.

Reply:
I moved this to be a function outside of the Test Case definition in the Simulation file. That way it just gets defined in there and you don't have to define it in the Default Test Case or the Template Test Case.